packages/backend/src/nest/socket/socket.service.ts

```diff
@@ -160,19 +160,6 @@ export class SocketService extends EventEmitter implements OnModuleInit {
         this.emit(SocketActionTypes.CONNECTION_PROCESS_INFO, ConnectionProcessInfo.REGISTERING_OWNER_CERTIFICATE)
       })
 
-      socket.on(SocketActionTypes.SAVE_OWNER_CERTIFICATE, async (payload: SaveOwnerCertificatePayload) => {
-        this.logger(`Saving owner certificate (${payload.peerId}), community: ${payload.id}`)
-
-        this.emit(SocketActionTypes.SAVED_OWNER_CERTIFICATE, payload)
```

Comment:
As I understand correctly, this SAVED_OWNER_CERTIFICATE is also not needed anymore because it's emitted in registerOwnerCertificate anyway?

Reply:
Yea